packages/ses-ava/CHANGELOG.md

```diff
@@ -3,6 +3,15 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [1.1.0](https://github.com/endojs/endo/compare/@endo/ses-ava@1.0.1...@endo/ses-ava@1.1.0) (2024-01-17)
+
+
+### Features
+
+* **env-options:** env-options conveniences for common cases ([#1710](https://github.com/endojs/endo/issues/1710)) ([4c686f6](https://github.com/endojs/endo/commit/4c686f6c9c3c54dbf73e8e7cd80a4dfebcbc61df))
```

Comment:
This one's not really a feature of ses-ava. It is a test of env-options that resides in ses-ava to avoid cyclic dependencies.

Reply:
I’m taking points away from `lerna` and giving them to `changesets`. (https://github.com/changesets/changesets)